besu/src/main/java/org/hyperledger/besu/cli/options/unstable/NativeLibraryOptions.java

```diff
@@ -53,7 +53,7 @@ public class NativeLibraryOptions {
           "Per default a native library is used for modexp. "
               + "If the Java implementation should be used instead, this option must be set to false",
       arity = "1")
-  private final Boolean nativeModExp = Boolean.TRUE;
+  private final Boolean nativeModExp = Boolean.FALSE;
```

Comment:
Why was this changed to `FALSE`? Shouldn't we prefer the native one per default?